### ISSUE TITLE
Menu access 

### ISSUE THREAD

**Issue by mabahj** [nolanlawson/Catlog#51]
On my HTC One (m7) with Cyanogenmod 12.1 Nightly 20150716 (Android 5.1.1), I'm unable to access the menu (loglevel, save, open etc)  in Catlog. My device does not have the old menu button and there is no onscreen alternative shown in Catlog.

**Comment by crackalak** [comment]
Have you tried to 'press and hold' on device home button to bring the menu up?

**Comment by mabahj** [comment]
Hi, 
Press and hold shows recent apps so that does not work for me.

**Comment by crackalak** [comment]
Not sure where the setting is on cyanogenmod , but on sense roms (I'm using insertcoin on my HTC m7) it's under Display, Gestures and Buttons, Home

You can change it so that press and hold brings up the old menu.

**Comment by mabahj** [comment]
Ok, thanks. That would work as a workaround, although I'd loose the home button functionality. 

Maybe the app should work without requiring a menu button? All my other apps do. 

(I could just find a different log-app, this is just a suggestion for further development.)

**Comment by crackalak** [comment]
The recent apps should then show on home button double tap so functionality isn't lost. 

I might be wrong but I think it's more of a case of the phone thinking that there is a hardware menu button so it doesn't show the menu as the 3 dots. 

This app isn't being developed any more by the original developer, but there are a few forks that have been changed for higher versions of Android. 
I did one myself quite a while back for the holo theme but it isn't fully rested, though I use it myself occasionally. This would fix the menu problem.

**Comment by Gitoffthelawn** [comment]
Is there a fork that is considered "mainstream" and used by many people?

**Comment by crackalak** [comment]
Not as far as I know, and looking at the charts, I'd say not unfortunately.

**Comment by Gitoffthelawn** [comment]
Too bad!  But thank you for responding!

**Comment by nolanlawson** [comment]
Yes, there is MatLog, which is material-inspired: https://github.com/plusCubed/matlog

**Comment by Gitoffthelawn** [comment]
@nolanlawson Thanks!  Although currently in beta, that certainly looks promising!

**Comment by mabahj** [comment]
Thanks, crackalak. I'll leave this bug open since all my other apps show the menu. But it looks like the htc one's menu button solution has been a problem in other apps as well, from what I read on the web,so the problem could be isolated to this device.
